application/classes/Ushahidi/Console/PostExporter.php

```diff
@@ -107,10 +107,10 @@ protected function executeExport(InputInterface $input, OutputInterface $output)
         $data = $this->data->get('search');
 
 		// Get CLI params
-		$limit = $input->getOption('limit', 100);
-		$offset = $input->getOption('offset', 0);
-		$job_id = $input->getOption('job', null);
-		$add_header = $input->getOption('include_header', true);
+		$limit = $input->getOption('limit') ? $input->getOption('limit') : 100;
```

Comment:
Note on this change: This was not working. Basically the default do not exist.GetOption accepts just the option name.

Reply:
oh. OK that sounds way more reasonable .